index.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const express = require('express');
+
+function startServer(config) {
+    const app = express();
+    const port = config.port || 3000;
+
+    app.use('/', express.static(path.normalize(`${config.currentDirectory}/${config.entry}`)));
+
+    app.listen(port, () => console.log(`Started server on port ${port}`));
```

Comment:
If you plan to keep that console.log(), there's a logger that skeletor-core passes in to the run() method.